src/pykeen/evaluation/__init__.py

```diff
@@ -1,6 +1,73 @@
 # -*- coding: utf-8 -*-
 
-"""Evaluation."""
+"""Evaluation.
+
+Knowlede graph embeddings models are usually evaluated based on link prediction, which is on knowledge graphs defined
+as predicting the tail/head entities for $(h,r)$/$(r,t)$ pairs.
+
+However, given the fact that usually true negative examples are not available, both the training and the test set
+contain only true facts. For this reason positive triples are corrupted to generate negative samples.
+For each test triple $t^{+} = (h,r,t) \in \mathcal{K}_{test}$ two sets of corrupted triples are constructed:
+
+    * $\mathcal{H}(r, t) = \{(h', r, t) \mid h' \in \mathcal{E} - \{h\}$
+    * $\mathcal{T}(h, r) = \{(h, r, t') \mid t' \in \mathcal{E} - \{t\}\}$
+
+where $\mathcal{H}(r, t)$ contains all the triples where the head entity has been corrupted, and $\mathcal{T}(h, r)$
+contains all the triples with corrupted tail entity.
+
+Among the corrupted triples in $\mathcal{H}(r, t)$ / $\mathcal{T}(h, r)$, there might be true triples that are part of
+the knowledge graph.
+If these false negatives are ranked higher than the current test triple $t^{+}$, the results might get distorted.
+Therefore, the \emph{filtered} evaluation setting has been proposed \cite{Bordes2013}, in which the corrupted triples
+are filtered to exclude known true facts from the train, validation and test set. Thus, the rank does not decrease when
+ranking another true entity higher.
+
+.. note ::
+    In PyKEEN, we provide training and evaluation pipelines for which the set of positive triples are pre-defined.
+    During hyper-parameter optimization (HPO), the set of triples that should be filtered out consists per default of
+    the training and validation set, and out of training, validation, and test triples during the evaluation
+    of the final model on the test set. We explicitly do not use test triples for filtering during HPO in order to avoid
```

Comment:
```suggestion
    the training and validation set. Only during the evaluation of the final model on the test set it consists of training, validation, and test triples. We explicitly do not use test triples for filtering during HPO in order to avoid
```